src/NServiceBus.Core/Recoverability/Faults/FaultMetadataExtractor.cs

```diff
@@ -24,26 +26,32 @@ public Dictionary<string, string> Extract(ErrorContext errorContext)
 
     static void SetExceptionMetadata(Dictionary<string, string> headers, Exception e)
     {
-        headers[FaultsHeaderKeys.ExceptionType] = e.GetType().FullName;
+        headers[FaultsHeaderKeys.ExceptionType] = e.GetType().FullName!;
 
         if (e.InnerException != null)
         {
-            headers[FaultsHeaderKeys.InnerExceptionType] = e.InnerException.GetType().FullName;
+            headers[FaultsHeaderKeys.InnerExceptionType] = e.InnerException.GetType().FullName!;
         }
 
-        headers[FaultsHeaderKeys.HelpLink] = e.HelpLink;
+        if (e.HelpLink is not null)
```

Comment:
It seems it [should be fine to not include the HelpLink](https://github.com/search?q=org%3AParticular+HelpLink&type=code) when it is not available. Overall this is also good because we have less headers to send around which trims down the message size.